.github/workflows/delete-review-app.yml

```diff
@@ -0,0 +1,92 @@
+name: Delete Review App
+
+on:
+  pull_request:
+    types: [closed]
+    branches: [main]
+
+jobs:
+  delete-review-app:
+    name: Delete Review App ${{ github.event.pull_request.number }}
+    if: github.actor != 'dependabot[bot]'
```

Comment:
We should probably change this condition. See: https://trello.com/c/wPYN3Buf

Reply:
Updated as per our chat on Friday